buildpipeline/pipeline.json

```diff
@@ -298,11 +298,13 @@
           }
         },
         {
-          "Name": "DotNet-CoreFx-Trusted-Windows-NoTest",
+          "Name": "DotNet-CoreFx-Trusted-Windows",
           "Parameters": {
             "PB_Platform": "x64",
```

Comment:
Why are we only changing the x64 build? Don't we need to do the same for x86 and arm? @weshaggard